Add Payment scheme

lagdush opened the pull request:
Hej @doublesz, @SUPERBOHATERN1  możecie sprawdzić czy wszystko gra? Rzućcie okiem czy warto dodawać ręcznie pole id do schematu.

lagdush committed:
Added payment scheme and payment model. Plus validation function with Joi.
 Changes to be committed:
	new file:   api/models/paymentSchema.js

doublesz commented on api/models/paymentSchema.js:
```diff
@@ -0,0 +1,75 @@
+  amount: {
+    type: Number,
+    min: 5,
+    max: 10000,
```
Tu chyba wartość max nie jest potrzebna

lagdush replied:
W sumie masz rację, warośc max nie jest potrzebna. Zaraz poprawię.

doublesz reviewed (commented):
Wszystko wygląda ok.
Jedyne co bym usunął to wartość max we właściwości amount.
No i trzeba wyedytować request, aby mergeować do development.

lagdush committed:
add tests to schema, add "jest": {"testEnvironment": "node" } to package.json, remove max value in amount in paymentScheme

 Changes to be committed:
	modified:   api/models/paymentSchema.js
	modified:   package.json
	new file:   test/paymentSchema.test.js

lagdush commented:
Dodałem testy i usunąłem wartość max

lagdush committed:
add expect and toBe in paymentSchema test

doublesz reviewed (approved):
Myślę, że wszystko jest jak należy.
Świetna robota z testami i walidacją!
